### PR TITLE
Spelling / Grammar

### PR DESCRIPTION
**Pull request by jsoref**
Mostly this is dropping doubled words. There's one brand fix.

## Detailed Description of the Pull Request / Additional comments

I'm close to shipping an [upgrade to check-spelling](https://github.com/check-spelling/check-spelling/releases/tag/v0.0.20-alpha5), but in testing it against this repository, I see one behavior change that I'd like to resolve. So, while I try to figure out how, I figured I'd at least leave the fixes submitted.

(I have some ideas, as I've been slowly working towards a solution to this problem, but it isn't a simple one line change.)

## Validation Steps Performed

[Ran check-spelling v0.0.20-alpha5](https://github.com/check-spelling/terminal/actions/runs/2100515961) w/ a refresh of the configuration based on https://github.com/check-spelling/spell-check-this/tree/prerelease/.github/workflows/

(It identified one minor bug which is fixed in 0.0.20-alpha6 - unreleased.)

### CODE REVIEW

File: src/host/CopyFromCharPopup.cpp [hunk 1]
```diff
@@ -15,7 +15,7 @@ CopyFromCharPopup::CopyFromCharPopup(SCREEN_INFORMATION& screenInfo) :
 }
 
 // Routine Description:
-// - This routine handles the delete from cursor to char char popup.  It returns when we're out of input or the user has entered a char.
+// - This routine handles the delete from cursor to char popup.  It returns when we're out of input or the user has entered a char.
```
Comment (jsoref) [inline src/host/CopyFromCharPopup.cpp:18]: Of the doubled words, this is the one about which I'm the least certain.

File: doc/specs/#5000 - Process Model 2.0/#1032 - Elevation Quality of Life Improvements.md [hunk 2]
```diff
@@ -456,7 +456,7 @@ accessible.
 
 Unfortunately, these issues are OS bugs that are largely out of our own control.
 We will continue to apply pressure to the centennial app team internally as we
-encounter these issues. They are are team best equipped to resolve these issues.
+encounter these issues. They are team best equipped to resolve these issues.
```
Comment (DHowett) [inline doc/specs/#5000 - Process Model 2.0/#1032 - Elevation Quality of Life Improvements.md:459]: This one might want to be "are the" :)

**Comment by DHowett** [comment]
I'm always unreasonably excited for check-spelling updates!

File: src/host/CopyFromCharPopup.cpp (hunk 1 above)
Comment (zadjii-msft) [reply src/host/CopyFromCharPopup.cpp:18]: this looks fine to me

**Review by zadjii-msft — COMMENTED** [review]
Yea I think the "are the" fix is the only one I'd like to see, but the rest is great! as the person who probably hits this bot the most often, thanks for the continuous updates here!

**Review by zadjii-msft — APPROVED** [review]

**Review by lhecker — APPROVED** [review]

**Pull request merged**